zellij-utils/src/setup.rs

```diff
@@ -23,11 +22,6 @@ pub fn find_default_config_dir() -> Option<PathBuf> {
         .flatten()
 }
 
-#[cfg(any(feature = "test", test))]
```

Comment:
This could potentially cause some confusion,
when I run the tests locally now I have one failing test,
since it sources the local configuration.

A possible solution could be starting with the `setup --clean`
in the makefile?

Reply:
Thanks for pointing this out!
Which test is failing for you? `cargo test` does not accept the arguments the same as a normal zellij binary, so I'm not sure how `setup --clean` would help.
For e2e tests, we can add it somewhere in https://github.com/zellij-org/zellij/blob/main/src/tests/e2e/remote_runner.rs but for unit tests we'll have to see.

Reply:
It is `try_from_cli_args_default()`, but now that I think more
about it that probably can get removed now.
I think It mattered more in the old test system.

Ah, I thought we could pass the flags through somehow,
since I think we do use the `--data-dir` flag in the tests atm?

Reply:
Are you sure we can remove it? Since it is used in a unit test, we can simply have it behind `#[cfg(test)]` attribute. We won't need a separate "test" feature.
The `--data-dir` is used in `build` command. For tests, `fn main()` is not the entry point and hence we cannot use zellij flags there.

Reply:
Ah, yes then it probably is still better to keep using
the distinction with `#[cfg(test)]`, if not for now.
I can see it being a point of confusion in the future
if personal configuration changes could affect the
outcome of tests.